src/main/routes/create.user.profile.ts

```diff
@@ -0,0 +1,39 @@
+import { fetchAll } from "../service/jurisdiction.service";
+import { createUserProfile } from "../service/create-user-service";
+import { UserProfile } from "../domain/userprofile";
+const router = require("../routes/home");
+
+/* GET create user form. */
+router.get("/createuser", (req, res, next) => {
+
+  fetchAll(req).then((response) => {
+    res.status(201);
+    const responseContent: { [k: string]: any } = {};
+    responseContent.jurisdictions = JSON.stringify(response);
+    if (req.session.error) {
+      responseContent.error = req.session.error;
+      delete req.session.error;
+    }
+    res.render("user-profiles/create-user-form", responseContent);
+  })
+    .catch((error) => {
+      // Call the next middleware, which is the error handler
+      next(error);
+    });
+});
+
+/* POST create user result. */
+router.post("/createuser", (req, res, next) => {
+  createUserProfile(req, new UserProfile(req.body.idamsId,
```

Comment:
Is it really called `idamsId`, or did you mean `idam`? (The system is actually known as IdAM, not "IdAMs".)